packages/web/src/components/gcds-pagination/gcds-pagination.css

```diff
@@ -0,0 +1,199 @@
+:host {
+  display: block;
+
+  ul {
+    padding: 0;
+    width: fit-content;
+    list-style: none;
+
+    li {
+      margin: var(--gcds-spacing-100);
+
+      a {
+        color: var(--var-pagination-default-text);
+        font-weight: 800;
```

Comment:
Can you use one of the existing font-weight tokens here? I don't think we are using `font-weight: 800;` anywhere else so maybe use 700 aka `--gcds-font-weights-bold` instead?